python/test/unit/language/test_block_pointer.py

```diff
@@ -25,7 +26,7 @@ def block_copy_kernel(a_ptr, b_ptr, N, BLOCK_SIZE: tl.constexpr, padding_option:
 ])
 def test_block_copy(dtypes_str, n, padding_option):
     capability = torch.cuda.get_device_capability()
-    if capability[0] >= 9:
```

Comment:
what's the point of adding this file if we are skipping the tests? Also block pointer is fully in sw on all targets now so not sure why it would work on hopper but not others.

Reply:
The purpose of this PR is to turn on this test on AMD gpu. I just do not want to change the logic on other backends. 

BTW, this if condition is to disable the test on hopper (checking line 29 and 30). If it works on everything, we should remove line 29 and 30.

Reply:
changed.